worker/rebuilder-nixos.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+set -x
+
+echo "Rebuilding $1"
+export PATH=/root/.nix-profile/bin:$PATH
+whoami
+
+DRV=$(cat $1 | grep StorePath | cut -d ":" -f 2)
```

Comment:
```suggestion
DRV=$(cat "$1" | grep StorePath | cut -d ":" -f 2)
```
Also most/all of the other variables should also have quotes around them (the unquoted behavior in shell can expand to 0..n words and is very counter intuitive).